src/PanelTraits/Create.php

```diff
@@ -92,6 +96,26 @@ public function getRelationFieldsWithPivot($form = 'create')
         });
     }
 
+    /**
+     * Create the relations for the current model.
+     *
+     * @param mixed $item The current CRUD model.
+     * @param array $data The form data.
+     * @param string $form Optional form type. Can be either 'create', 'update' or 'both'. Default is 'create'.
```

Comment:
That either sounds odd. "one of" is more idiomatic (either implies just two to me).

Reply:
I copy-pasted this from the existing code. I'm okay to change this everywhere, though. :)

Reply:
Hmmm....not sure about an everywhere change.

Reply:
I thought about doing a PR in the near future to go through all the docs. Some are missing, some are incomplete, some are using different styles. Leave it like this for now and do it then?